test/pathname/test_pathname.rb

```diff
@@ -289,7 +289,7 @@ def relative?(path)
   end
 
   def relative_path_from(dest_directory, base_directory)
-    Pathname.new(dest_directory).relative_path_from(Pathname.new(base_directory)).to_s
```

Comment:
I lost track of this PR

I changed this to _not_ pass base_directory to Pathname.new, because then none of the tests are actually calling relative_path_from with a non-Pathname instance.

Do you want tests that call relative_path_from with both examples of Pathname _and_ String arguments ?